portal/app/[locale]/tunnel/_hooks/useWaitTime.ts

```diff
@@ -0,0 +1,22 @@
+import { useQuery } from '@tanstack/react-query'
+import { ToEvmWithdrawOperation } from 'types/tunnel'
+import {
+  getTimeToProveInSeconds,
+  getTimeToFinalizeInSeconds,
+} from 'utils/wait-times/evmWithdrawals'
+
+export const useEvmWithdrawTimeToProve = (withdrawal: ToEvmWithdrawOperation) =>
+  useQuery({
+    enabled: !!withdrawal,
+    queryFn: () => getTimeToProveInSeconds(withdrawal),
+    queryKey: ['time-to-prove-withdraw', withdrawal.transactionHash],
+  })
+
+export const useEvmWithdrawTimeToFinalize = (
+  withdrawal: ToEvmWithdrawOperation,
+) =>
+  useQuery({
+    enabled: !!withdrawal,
+    queryFn: () => getTimeToFinalizeInSeconds(withdrawal),
+    queryKey: ['time-to-finalize-withdraw', withdrawal.transactionHash],
+  })
```

Comment:
I think we need a `refetchInterval` here for both. It's not needed to check every block, but perhaps every 2 blocks? (That would be 24 seconds, as each block is ~12 seconds)

Reply:
added